### PR TITLE
Python3 support

### PR DESCRIPTION
**Pull request by jpic**
I found out this was blocking the release of DAL v3 ^^

### CODE REVIEW

**Comment by jpic** [comment]
```
  py27-django18: commands succeeded
  py27-django19: commands succeeded
  py34-django18: commands succeeded
  py34-django19: commands succeeded
  congratulations :)
```

;)

File: tests/test_not_implemented.py [hunk 1]
```diff
@@ -61,7 +63,7 @@ class NotImplementedMeta(MetaTestGenerator):
     EXPECTED_EXCEPTION = NotImplementedError
 
 
-class TestNotImplemented(TestCase):
+class TestNotImplemented(six.with_metaclass(NotImplementedMeta, TestCase)):
     """Test methods that are not implemented and should raise NotImplemented."""
     __metaclass__ = NotImplementedMeta
```
Comment (clokep) [inline tests/test_not_implemented.py]: I think you can get rid of `__metaclass__` when using `six.with_metaclass`.

**Comment by clokep** [comment]
Looks pretty sane, I'll give a more thorough review and do a release today or tomorrow.

File: tests/test_not_implemented.py (hunk 1 above)
Comment (jpic) [inline tests/test_not_implemented.py]: Not for python2 support during my tests.

**Comment by clokep** [comment]
Seems to work pretty! I have a couple of comments I'll fix when merging this! Thanks!

**Comment by jpic** [comment]
Thx !

FTR, it turned out using 2to3 to patch the files and then make it Python2-compatible was the easiest way, and mental-sanity preserving way !

**Comment by clokep** [comment]
I merged this manually on the command line, thanks!

**Comment by clokep** [comment]
I pushed a 0.4 with this included: https://pypi.python.org/pypi/django-querysetsequence/0.4

**Comment by jpic** [comment]
You rock, thanks !​

**Pull request closed without merging**